### PR TITLE
Try to fix tippy in explore page

### PR DESCRIPTION
**Pull request by baseplate-admin**
image

image

### CODE REVIEW

**Comment by baseplate-admin** [comment]
This problem is weird

**Comment by tokitou-san** [comment]
> This problem is weird

Indeed, seems like an issue with position: `left`

**Review by tokitou-san — APPROVED** [review]
LGTM

File: src/tippy.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 .tippy-box {
     @apply bg-transparent;
+    max-width: none !important;
```
Comment (tokitou-san) [inline src/tippy.scss:7]: https://github.com/atomiks/tippyjs/issues/451

Comment (baseplate-admin) [reply src/tippy.scss:7]: How tf did you stumble on that Issue?

**Pull request merged**